getting_started_rosa/quickstart-rosa.adoc

```diff
@@ -0,0 +1,12 @@
+[id="quickstart-rose"]
+= Getting started with {product-title}
+include::modules/common-attributes.adoc[]
+:context: quickstart-rosa
+
+toc::[]
+
+This guide walks through setting up your first {product-title} cluster using `rosa`, the {product-title} command line utility.
```

Comment:
s/This guide walks through/This guide walks you through/
s/command line utility/command-line utility/

Reply:
Done.